widget/list.go

```diff
@@ -23,7 +23,8 @@ var _ fyne.Focusable = (*List)(nil)
 
 // List is a widget that pools list items for performance and
 // lays the items out in a vertical direction inside of a scroller.
-// List requires that all items are the same size.
+// By default List requires that all items are the same size, but specific
```

Comment:
This is more grammatically correct:
```suggestion
// By default, List requires that all items are the same size, but specific
```

Reply:
Ah, yeah that's better